src/breakpoints.jl

```diff
@@ -96,6 +112,7 @@ function breakpoint(file::AbstractString, line::Integer, condition::Condition=no
     add_to_existing_framecodes(bp)
     idx = findfirst(bp2 -> same_location(bp, bp2), _breakpoints)
     idx === nothing ? push!(_breakpoints, bp) : (_breakpoints[idx] = bp)
+    firehooks(breakpoint, bp)
```

Comment:
Should this if it is replacing an existing one,
also do 
`firehooks(remove, bp)` ?

Reply:
That seems like a good idea.